fvm/handler/contract.go

```diff
@@ -98,14 +100,37 @@ func (h *ContractHandler) RemoveContract(address runtime.Address, name string, s
 	return nil
 }
 
+type contractUpdateList []programs.ContractUpdate
+
+func (l contractUpdateList) Len() int      { return len(l) }
+func (l contractUpdateList) Swap(i, j int) { l[i], l[j] = l[j], l[i] }
+func (l contractUpdateList) Less(i, j int) bool {
+	switch bytes.Compare(l[i].Address[:], l[j].Address[:]) {
+	case -1:
+		return true
+	case 0:
+		return l[i].Name < l[j].Name
+	default:
+		return false
+	}
+}
+
 func (h *ContractHandler) Commit() ([]programs.ContractUpdateKey, error) {
 
 	h.lock.Lock()
 	defer h.lock.Unlock()
 
 	updatedKeys := h.UpdateKeys()
+	updateList := make(contractUpdateList, 0)
+
+	for _, uk := range h.draftUpdates {
+		updateList = append(updateList, uk)
+	}
+	// sort does not need to be stable as the contract update key is unique
+	sort.Sort(updateList)
```

Comment:
maybe just adding a test to make sure it works as expected.

Reply:
added test [62d68d8](https://github.com/onflow/flow-go/pull/2003/commits/62d68d8bfd7d90ec21c257ccc60a0bd722db57ec)